Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -63,15 +63,15 @@ public extension MoyaProvider {
             }
 
             let networkCompletion: Moya.Completion = { result in
-              if self.trackInflights {
-                self.inflightRequests[endpoint]?.forEach { $0(result) }
+                if self.trackInflights {
```

Comment:
Why are these spaces here? Am I missing something?

Reply:
I guess its just indentation fixing from when I re-idented the file after working on it.